examples/next/pages/ui/hooks/useDataStore/models/index.js

```diff
@@ -0,0 +1,7 @@
+// @ts-check
```

Comment:
Not a super big deal and i missed this earlier but should we just convert this to a TS file?

Reply:
Correct me if I am wrong but I believe everything in the models folder is generated by codegen.